code/modules/food_and_drinks/restaurant/customers/_customer.dm

```diff
@@ -84,3 +84,27 @@
 		var/mutable_appearance/flag = mutable_appearance(customer.icon, "french_flag")
 		flag.appearance_flags = RESET_COLOR
 		return flag
+
+
+
+/datum/customer_data/japanese
+	nationality = "Space-Japanese"
+	prefix_file = "strings/names/japanese_prefix.txt"
+	base_icon = "japanese"
+	clothing_sets = list("japanese_animes")
+
+	found_seat_lines = list("Konnichiwa!", "Arigato gozaimasuuu~", "I hope there's some beef stroganoff...")
+	cant_find_seat_lines = list("I want to sit under the cherry tree already, senpai!", "Give me a seat before my Tsundere becomes Yandere!", "This place has less seating than a capsule hotel!")
+	leave_mad_lines = list("I can't believe you did this! WAAAAAAAAAAAAAH!!", "I-It's not like I ever wanted your food! B-baka...", "I was gonna show you my tip!")
+	leave_happy_lines = list("Oh CHEF! This is the happiest day of my life. I love you!", "I take a potato chip.... AND EAT IT!", "Itadakimasuuu~", "Gochisousama desu!")
```

Comment:
these lines are not specific to the chef, I know I made some lines that are not really agnostic, but I'd like to avoid adding more if possible.